clang/test/Sema/enum.c

```diff
@@ -1,23 +1,23 @@
-// RUN: %clang_cc1 -triple %itanium_abi_triple %s -fsyntax-only -verify -pedantic
+// RUN: %clang_cc1 -triple %itanium_abi_triple %s -fsyntax-only -verify=expected,c99 -pedantic
```

Comment:
Should this get `-std=c99`?

Reply:
Hmm, I think removing the prefix is more accurate to make the intent more clear. I don't know why I named it c99, the idea was to test not C23. I renamed it accordingly.